### PR TITLE
feat: Remove Custom Token Name

### PR DESCRIPTION
**Pull request by karelianpie**
## Motivation and Context

Users have the ability to designate a custom GIVE Token name, this causes some confusion for users when all of our docs refer to GIVE.

## Description

* Remove the custom token name field from the Circle Admin general section if it is not custom, expands remaining fields to take full width;
* Disable the custom token name field if there is one already set;
* Throw an error if the user somehow manages to attempt to change the token name.

## Test and Deployment Plan

FE: Ran locally with and without a custom token name and saw the diff
BE: Tried to set a custom token name and saw the error message, users will not be able to do that from the frontend as the field is disabled.

Refer to the screenshots below

### With a custom token name

Screen Shot 2022-09-19 at 2 52 19 pm

Screen Shot 2022-09-19 at 2 52 49 pm

### Without a custom token name

Screen Shot 2022-09-19 at 2 55 55 pm

### When trying to set a new custom name

Screen Shot 2022-09-19 at 2 58 10 pm

## Reviewers

@topocount 

## Related Issue

Closes https://github.com/coordinape/coordinape/issues/1335

### CODE REVIEW

File: api/hasura/actions/_handlers/updateCircle.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { VercelRequest, VercelResponse } from '@vercel/node';
 
 import { authCircleAdminMiddleware } from '../../../../api-lib/circleAdmin';
 import { endNominees, updateCircle } from '../../../../api-lib/gql/mutations';
+import * as queries from '../../../../api-lib/gql/queries';
```
Comment (crabsinger) [inline api/hasura/actions/_handlers/updateCircle.ts:5]: can we just import getCircle and not the entire `*` ?

Comment (karelianpie) [reply api/hasura/actions/_handlers/updateCircle.ts]: Refactored with https://github.com/coordinape/coordinape/pull/1360/commits/3793e52a6ff515e4f93eb86940ff8e2763148fd6

**Comment by crabsinger** [comment]
@karelianpie please rebase

(did you happen to mark this PR to allow maintainers to edit it?) https://github.com/karelianpie/coordinape/blob/issue-1335/CONTRIBUTING.md#pr-guidelines I'm curious because this allows me to rebase the PR myself...

**Comment by karelianpie** [comment]
> @karelianpie please rebase
> 
> (did you happen to mark this PR to allow maintainers to edit it?) https://github.com/karelianpie/coordinape/blob/issue-1335/CONTRIBUTING.md#pr-guidelines I'm curious because this allows me to rebase the PR myself...

Rebased just now, I've been trying to keep it in sync with main (this was my 3rd rebase) but a lot is happening on `main` :)

I'm not sure, I think so -- feel free to rebase next time so the PR review cycle is a bit faster, I'll try my best to keep it in sync but `main` has been busy today

**Review by levity — APPROVED** [review]

**Comment by levity** [comment]
@karelianpie the same Cypress test has failed twice; can you please look into it?

**Comment by karelianpie** [comment]
> @karelianpie the same Cypress test has failed twice; can you please look into it?

https://github.com/coordinape/coordinape/pull/1360/commits/3237fab0a578811039843767191fde65b4f27dcc fixed it, thanks @levity

**Review by levity — APPROVED** [review]

**Pull request merged**